service/assessment/assessment_test.go

```diff
@@ -1038,8 +1039,11 @@ func TestHandleEvidence(t *testing.T) {
 				resourceId: "my-resource-id",
 			},
 			wantErr: func(t assert.TestingT, err error, i ...interface{}) bool {
-				assert.Contains(t, err.Error(), "could not get stream to evidence store")
-				return true
+				if !assert.NotEmpty(t, err) {
```

Comment:
```suggestion
				if assert.Empty(t, err) {
```

Reply:
This changes the semantic (and makes the test fail). We want to assert that the error is NOT empty. And only return false if it is actually empty.